lib/install/management/virtual_app.go

```diff
@@ -79,7 +79,7 @@ func (d *Dispatcher) createVApp(conf *config.VirtualContainerHostConfigSpec, set
 			Name:      "vSphere Integrated Containers",
 			Vendor:    "VMware",
 			VendorUrl: "http://www.vmware.com/",
-			Version:   "0.0.1",
+			Version:   "0.5.5",
```

Comment:
It seems to work nice for me. Not sure where the version info is set explicitly though. Or it might be set automatically from a git tag.. Anybody know about this?